webapp/portlet/src/main/webapp/WEB-INF/gatein-resources.xml

```diff
@@ -371,6 +371,9 @@
     <depends>
       <module>commonVueComponents</module>
     </depends>
+    <depends>
+      <module>documentPreview</module>
```

Comment:
Can you confirm that this js is located in social ? I guess it is in ECMS
In meeds, without ECMS, it will make a problem